libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/saveDashboardHandler.ts

```diff
@@ -170,31 +171,39 @@ export function* saveDashboardHandler(
     ctx: DashboardContext,
     cmd: SaveDashboard,
 ): SagaIterator<DashboardSaved> {
-    const saveCtx: SagaReturnType<typeof createDashboardSaveContext> = yield call(
-        createDashboardSaveContext,
-        cmd,
-    );
-    const isNewDashboard = saveCtx.persistedDashboard === undefined;
-    let result: DashboardSaveResult;
-
-    if (isNewDashboard) {
-        result = yield call(save, ctx, saveCtx, createDashboard, "@@GDC.DASH.SAVE_NEW");
-    } else {
-        result = yield call(save, ctx, saveCtx, updateDashboard, "@@GDC.DASH.SAVE_EXISTING");
-    }
-
-    const { dashboard, batch } = result;
-
-    yield put(batch);
-
-    if (isNewDashboard) {
-        yield setContext({
-            dashboardContext: {
-                ...ctx,
-                dashboardRef: dashboard.ref,
-            },
-        });
+    try {
+        yield put(dashboardCommandStarted(ctx, cmd));
```

Comment:
same as saveAs - perhaps better in root command handler?

Reply:
Good point, fixed in 0652b73